layouts/shortcodes/see-also.html

```diff
@@ -1,18 +1,19 @@
+{{ $save := . }}
 {{ $page := .Get "page" }}
 {{ $anchor := .Get "anchor" }}
-{{ $save := . }}
-{{ if not ( $.Site.GetPage $page ) }}
-{{ if (eq $.Site.Params.linkErrorLevel "WARNING") }}
-{{ warnf "[see-also] REF_NOT_FOUND: Ref \"%s\": \"%s\": page not found" $page $save.Page.File.Path }}
-{{ else }}
-{{ errorf "[see-also] REF_NOT_FOUND: Ref \"%s\": \"%s\": page not found" $page $save.Page.File.Path }}
-{{ end }}
-{{ end }}
+{{ $prefix :=  or ($save.Get "prefix") "See Also"  }}
+
 {{ with $.Site.GetPage $page }}
-{{ $title := $save.Get "title" }}
-{{ if $title }}
-{{ printf "<p><em>See Also:</em> <a href=%s#%s>%s</a></p>" (ref . $page) $anchor $title | safeHTML }}
+    {{ $title := or ($save.Get "title") .Title }}
+    {{ $url := ref . $page }}
+    {{ with $anchor }}
+        {{ $url = print $url "#" $anchor }}
+    {{ end }}
+    <p><em>{{ $prefix }}</em>: <a href="{{ $url }}">{{ $title }}</a></p>
```

Comment:
@abbyad - soooo... this indent looks pretty, and would be my preferred formatting, however if we use a `{{<` to including the shortcode like we do in the style guide, all good - everything works. However, in some places, and this tripped me up a whole lot in development, when we use `{{%` to include the shortcode, it actually renders the HTML:

![inline code](https://user-images.githubusercontent.com/8253488/87076368-819e0380-c1d6-11ea-821e-c8b5c14576bf.png)

I finally traced this back to the [0.55.0 release](https://github.com/gohugoio/hugo/releases/tag/v0.55.0) changing how they render short codes (I guess no more [blackfriday](https://gohugo.io/getting-started/configuration-markup)?).

tl;dr - we have 7 that use the `{{<` and 29 that use `{{%` - so I guess we should not indent so it works with both styles of code?

Reply:
Ah, that helps to clear this up. Can you think of a reason to use `{{%` for this shortcode? Seems as though we should use `{{<` over `{{%`, and update the style guide and existing files -- you said you like `sed`, right? :)

Reply:
The only reason I can think to use `{{%` is that it's a documented and accepted syntax and "because it formats things better" doesn't seem like _that_ big of a reason to not use.  

However, when we have thousands of documents in this repo, and a lot more of them are templates....maybe formatting becomes more important?!  So, yeah, like you `sed` I `sed` (big fan of puns ;), let's go for it! I'll swap 'em out in this PR - stay tuned!